johnny/index.html

```diff
@@ -0,0 +1 @@
+<div>hello world</div>
```

Comment:
do you think div is the correct HTML element to use here?

Reply:
maybe a div makes sense here. what do you think @JohnnyTTran ?